tvb_build/Jenkinsfile-Step1

```diff
@@ -38,6 +38,9 @@ pipeline {
                     bash install_full_tvb.sh
                     python build_step1.py
                 '''
+                withCredentials([usernamePassword(credentialsId: 'tvb-revision', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USERNAME')]) {
+                        sh('git commit scientific_library/tvb/basic/config/tvb.version -m "Update GIT revision number automatically from Jenkins" && git push https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/the-virtual-brain/tvb-root.git HEAD:tvb-version || true')
+                 }
```

Comment:
Branch name will need to be changed into `master`, or can we take it from the env to be the current branch ?

Reply:
After the current changes we are taking the branch name automatically.